data/events.json

```diff
@@ -121,8 +121,8 @@
     "StartingDate": "2023-12-02",
     "EndingDate": "2023-12-02",
     "Venue": "TBD (Visakhapatnam, India)",
```

Comment:
@CharanMN7 please update the Venue as "GITAM University, Visakhapatnam"

Reply:
updated the venue to "GITAM University, Visakhapatnam" 👍